collections-arrays/RemoveDuplicatesFromAnArray/BenchmarkRunner/RemoveDuplicateElementsBenchmark.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BenchmarkDotNet.Attributes;
+using RemoveDuplicatesFromAnArray;
+
+namespace BenchmarkRunner
+{
+    [RankColumn]
+    [MemoryDiagnoser]
+
+    public class RemoveDuplicateElementsBenchmark
+    {
+        public static readonly RemoveDuplicateElements _duplicatesRemoval = new RemoveDuplicateElements();
```

Comment:
Why static? Should be private. Also when declaring public properties, naming convention is different (capital letter)